shared/cloud/ess-getting-started.asciidoc

```diff
@@ -14,6 +15,7 @@ There's no faster way to get started than with our hosted {ess} on Elastic Cloud
 
 . Select a solution and give your deployment a name.
 
-. Click *Create deployment* and copy the password for the `elastic` user and Cloud ID information.
+. Click *Create deployment* and copy the password for the `elastic` user.
```

Comment:
> copy the password

@nrichers  Super minor nit, but should this be "download the password" to align with the cloud UI?

<img width="528" alt="Screen Shot 2021-04-02 at 4 44 58 PM" src="https://user-images.githubusercontent.com/40268737/113452812-cc018900-93d2-11eb-9518-bde7d38c6a23.PNG">

Reply:
I'll just change it out. Feel free to comment if that's the wrong call.